test/fixtures/whitespace_in_value.ftl

```diff
@@ -0,0 +1,10 @@
+# Caution, lines 4 and 6 contain white-space-only lines
```

Comment:
Looking at the diff, aren't they lines 6 and 7?

Reply:
![image](https://user-images.githubusercontent.com/265818/44193721-0f306000-a134-11e8-82dc-70f9497d6c04.png)